app/styles/_forms.less

```diff
@@ -7,6 +7,10 @@
   vertical-align: -2px;
 }
 
+// Generic class to wrap a single form element row that contains multiple actions separated by a divider
+.form-group-actions .btn {
+  padding: 0;
```

Comment:
We want to target `btn-link` specifically

Reply:
yep. good catch